packages/frontend/src/app/(side-nav)/data-availability/summary/_components/ethereum-da-entry.tsx

```diff
@@ -15,56 +16,59 @@ export function EthereumDaEntry({ entry }: { entry: DaSummaryEthereumEntry }) {
   const tvsToFormat = entry.usedIn.length > 0 ? entry.tvs : 0
 
   return (
-    <div className="mb-3 overflow-auto max-md:-mr-4">
-      <div className="relative flex min-w-[940px] items-center justify-between gap-3 rounded-lg bg-blue-300 px-4 py-1 dark:bg-blue-950 max-md:mr-4 md:py-3">
-        <Tooltip>
-          <TooltipTrigger>
-            <InfoIcon variant="blue" />
-          </TooltipTrigger>
-          <TooltipContent className="z-[1000]">
-            From the rollup perspective, Ethereum&apos;s canonical chain cannot
-            contain unavailable data commitments as full nodes self-verify the
-            data availability of each block, discarding blocks with unavailable
-            data. The rollup state validating bridge has access to all the data,
-            as it is posted on chain.
-          </TooltipContent>
-        </Tooltip>
-        <div className="flex w-full items-center justify-between ">
-          <div className="flex items-center justify-center gap-2">
-            <Image
-              className="min-h-[20px] min-w-[20px]"
-              src={`/icons/${entry.slug}.png`}
-              width={16}
-              height={16}
-              alt={`${entry.name} logo`}
-            />
-            <ProjectNameCell project={entry} />
-          </div>
-          <div className="flex gap-2">
-            <PropertyLabel>TVS</PropertyLabel>
-            <div className="text-sm font-medium leading-[14px]">
-              {formatCurrency(tvsToFormat, 'usd')}
+    <Link href={entry.href ?? ''} className="group">
+      <div className="mb-3 overflow-auto max-md:-mr-4">
+        <div className="relative flex min-w-[940px] items-center justify-between gap-3 rounded-lg bg-blue-300 px-4 py-1 transition-colors duration-200 group-hover:bg-blue-400 dark:bg-blue-950 group-hover:dark:bg-blue-900 max-md:mr-4 md:py-3">
+          <Tooltip>
+            <TooltipTrigger>
+              <InfoIcon variant="blue" />
+            </TooltipTrigger>
+            <TooltipContent className="z-[1000]">
+              From the rollup perspective, Ethereum&apos;s canonical chain
```

Comment:
Quite weird that this is here and not in the config

Reply:
changed, now part of ethereum da config